tests/test-sources/plugins/utils/harpoon.nix

```diff
@@ -15,6 +15,7 @@
     plugins.harpoon = {
       enable = true;
 
+      enableTelescope = true;
```

Comment:
You are going to need to enable telescope in that test, else the assertion will trigger :P

Reply:
yeh i disabled it but maybe it makes more sense to test with it enabled